go-controller/pkg/node/openflow_manager_test.go

```diff
@@ -0,0 +1,66 @@
+package node
+
+import "testing"
+
+func TestOpenFlowManagerDefaultNetOVSBridgeFinder(t *testing.T) {
+	testCases := []struct {
+		name                  string
+		desc                  string
+		inputPortInfo         string
+		expectedBridgeName    string
+		expectedPatchPortName string
+	}{
+		{
+			name:                  "empty input ports",
+			inputPortInfo:         "",
+			expectedBridgeName:    "",
+			expectedPatchPortName: "",
+		},
+		{
+			name:                  "input ports without patch ports",
+			inputPortInfo:         "port1",
+			expectedBridgeName:    "",
+			expectedPatchPortName: "",
+		},
+		{
+			name: "input ports with a patch port",
+			inputPortInfo: `
+port1
+port2
+patch-br-ex_multi-homing-worker-0.maiqueb.org-to-br-int`,
+			expectedBridgeName:    "br-ex",
+			expectedPatchPortName: "patch-br-ex_multi-homing-worker-0.maiqueb.org-to-br-int",
+		},
+		{
+			name: "input ports with a patch port for a localnet network",
+			inputPortInfo: `
+port1
+port2
+patch-vlan2003_ovn_localnet_port-to-br-int`,
+			expectedBridgeName:    "",
+			expectedPatchPortName: "",
+		},
+		{
+			name: "input ports with a patch port for the default network and a localnet",
+			inputPortInfo: `
+port1
+port2
+patch-vlan2003_ovn_localnet_port-to-br-int
+patch-br-ex_multi-homing-worker-0.maiqueb.org-to-br-int`,
+			expectedBridgeName:    "br-ex",
+			expectedPatchPortName: "patch-br-ex_multi-homing-worker-0.maiqueb.org-to-br-int",
```

Comment:
related to this pr but slightly different dimension.. but I was also hitting:
```
F0802 02:59:29.051867   23805 ovnkube.go:136] failed to run ovnkube: failed to start node network controller: failed to start default node network controller: failed to bootstrap OVS flows: failed to get flows on bridge "breth0_l3.network":, stderr: "ovs-ofctl: breth0_l3.network is not a bridge or a socket\n", error: exit status 1
```
in my PR for OPFM, my understanding is @dceara  did https://github.com/ovn-org/ovn-kubernetes/pull/4476/commits/bbb476b27cc32c74d2df686f3a7f0bd2d0889b5b

wondering if that will also be still needed ...I guess so? given we are only thinking about localnet harcoded based parsing...

Reply:
just wondering if there are broader set of "to be excluded ports" and if the reverse is easier i.e allowlisting the known ports for which we need to setup basic flows rather than excluding an unknown list?

Reply:
well, we can read from the configuration and know which bridge is supposed to be the right one.

People will create all kinds of patch ports in their system ... IMHO we should remove the entire regex shenanigan and instead just read from the config.